arch/cortex-m/src/mpu.rs

```diff
@@ -472,7 +472,7 @@ impl<const NUM_REGIONS: usize, const MIN_REGION_SIZE: usize> mpu::MPU
                 let tz = start.trailing_zeros();
                 if tz < 32 {
                     // Find the largest power of two that divides `start`
-                    (1 as usize) << tz
+                    1_usize << tz
```

Comment:
I acknowledge that this is presumably the more idiomatic Rust, and fine, whatever, we can write this way, but wow this is much less clear to my eyes.

Reply:
I think it's the underscore that makes this seem weird to me. Would it be more clear without the underscore @ppannuto?

```rust
    1usize << tz
```

versus

```rust
    1_usize << tz
```

Reply:
But we should really be consistent. So if `0x8000000_u32` and `0b01_u32` seem like the clear way to write them (and I think the underscore _is_ better there), then `1_usize` is for the best.